dali/operators/generic/pad.cc

```diff
@@ -23,8 +23,9 @@ namespace dali {
 
 DALI_SCHEMA(Pad)
   .DocStr(R"code(Pads all samples with `fill_value` in the given `axes`,
-to match the size of the biggest dimension on those axes in the batch.
-The element padding axes is specified with the argument `axes`.
+to match the biggest extent in the batch for those axes, or to match the minimum `shape`
+specified.
+
```

Comment:
Does the bellow example require update?

Reply:
Not required, but might be a good idea to enhance those examples to demonstrate the new behavior as well